.ci/pipeline/docs.yml

```diff
@@ -16,26 +16,47 @@ pr:
     - doc
     - examples
 
+variables:
+  - name: python.version
+    value: '3.7'
+  - name: daal4py.version
+    value: '2021.2.0'
+
 pool:
   vmImage: 'ubuntu-20.04'
-
 steps:
-- script: |
-    cd doc
-    sudo apt-get install python3-setuptools
-    sudo pip3 install -r requirements.txt
-  displayName: 'Install requirements'
-
-- script: |
-    cd doc
-    make html
-  displayName: 'Build documentation'
-
-- script: |
-    cp -R doc/_build $(Build.ArtifactStagingDirectory)/html
-  displayName: 'Copy build'
-
-- task: PublishPipelineArtifact@0
-  inputs:
-    artifactName: 'daal4py-documentation'
-    targetPath: '$(Build.ArtifactStagingDirectory)/html'
+  - task: UsePythonVersion@0
+    displayName: 'Use Python $(python.version)'
+    inputs:
+      versionSpec: '$(python.version)'
+  - script: |
+      conda update -y -q conda
+      conda create -q -y -n CB -c intel python=$(python.version) daal-devel
+    displayName: 'Conda create'
+  - script: |
+      . /usr/share/miniconda/etc/profile.d/conda.sh
+      conda activate CB
+      pip install -r requirements-dev.txt
+      pip install -r requirements-doc.txt
+    displayName: 'Install requirements'
+  - script: |
+      . /usr/share/miniconda/etc/profile.d/conda.sh
+      conda activate CB
+      export DALROOT=$CONDA_PREFIX
+      export PKG_VERSION=$(daal4py.version)
+      export NO_DIST=1
+      python setup.py install --single-version-externally-managed --record=record.txt
```

Comment:
Just as alternative option - call docbuild after regular daal4py build instead of building daal4py separately?

Reply:
Can we set up a trigger for a sub-task? Or always run the rules?

Reply:
yes - we can setup triggers

Reply:
docs build and attach for 5-10 sec. I think - we don't need a trigger :)